lib/src/room.dart

```diff
@@ -2090,27 +2090,38 @@ class Room {
   /// Returns the [PushRuleState] for this room, based on the m.push_rules stored in
```

Comment:
Would you please update the comment as well :)

Reply:
To what? It still does exactly the same 🤔